platform/Stream.cpp

```diff
@@ -112,14 +103,14 @@ void Stream::rewind() {
 }
 
 int Stream::isatty() {
-    return 0;
+    return 1;
```

Comment:
Yes, planning to deprecate. The issue is that fdopen() (and the C library if it's working) checks the isatty() to determine whether to set non-buffered.

I guess we could leave this unfixed, and put back the manual set_unbuffered_stream in the constructor, given that this is on the way out. I see the argument for not fixing to-be-deprecated things. Just as we're not touching the broken/unhelpful read/write calls here.